release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2832,6 +2825,107 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the installation program attempted to create a private zone for a cluster that needed to be installed on a {gcp-full} shared virtual private network (VPC). This caused the installation of the cluster to fail. With this release, a fixed skips the creation of the private zone so that this cluster installation issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-42142[*OCPBUGS-42142*])
+
+* Previously, when the installation program installed a cluster on a {gcp-full} VPC, role bindings for the control plane service account were not removed by the program. With this release, a fix removes the role bindings so that your cluster no longer includes these artifacts. (link:https://issues.redhat.com/browse/OCPBUGS-42116[*OCPBUGS-42116*])
+
+* Previously, if you enabled on-cluster layering for your cluster and you attempted to configure kernal arguments in the machine configuration, machine config pools (MCPs) and nodes entered a degraded state. This happened because of a configuration mismatch. With this release, a check for kernel arguments for a cluster with OCL-enabled ensures that the arguments are configured and applied to nodes in the cluster. This update prevents any mismatch that previously occured between the machine configuration and the node configuration. (link:https://issues.redhat.com/browse/OCPBUGS-42008[*OCPBUGS-42008*])
+
+* Previously, creating cron jobs to create pods for your cluster caused the component that fetches the pods to fail. Because of this issue, the *Topology* page on the {product-title} web console failed. With this release, a `3` second delay is configured for the component that fetches pods that are generated from the cron job so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-41685[*OCPBUGS-41685*])
+
+* Previously, when you deployed an {product-title} cluster that listed a `TechPreviewNoUpgrade` feature gate in its configured on {rh-openstack}, the `cluster-capi-operator` pod crashed. This occured because the Cluster CAPI Operator expected a different API version than the one that was served. With this release, an update to the Cluster CAPI Operator ensures the Operator uses the correct version of the API so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-41576[*OCPBUGS-41576*])
+
+* Previously on Red{nbsp}Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP), a cluster that used mirroring release images migth result in existing node pools to use the hosted cluster OS version instead of the NodePool version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-41552[*OCPBUGS-41552*])
```

Comment:
```suggestion
* Previously on Red{nbsp}Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP), a cluster that used mirroring release images might result in existing node pools to use the hosted cluster operating system version instead of the `NodePool` version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-41552[*OCPBUGS-41552*])
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_o